src/containers/Banner.tsx

```diff
@@ -205,8 +218,20 @@ const Banner = ({ isEditBanner, user }: BannerProps) => {
               )}
             </ChangePhotoDiv>
 
-            {banner && (
-              <img className="img-fluid w-100 h-100" src={banner} alt="" />
+            {temporaryBanner && isEditBanner ? (
+              <img
+                className="img-fluid w-100 h-100"
+                src={temporaryBanner}
+                alt=""
+              />
+            ) : (
+              banner && (
+                <img
+                  className="img-fluid w-100 h-100"
+                  src={`data:image/png;base64,${banner}`}
+                  alt=""
```

Comment:
I don't wont delete alt="" because i have warning in eslint -  "img elements must have an alt prop..."
In the same time in AvatarImg we haven't this warning.

Reply:
ok